test/Client/Endpoint/DataRetentionEndpointTest.php

```
@@ -0,0 +1,155 @@
+    }
+
+    #[Test]
+    public function getTeamsForRetentionPolicyBuildsCorrectRequest(): void
+    {
+        $this->mockJsonResponse(200, [['status' => 'ok']]);
+
+        $policy_id = 'test-policy_id';
+        $page = 1;
+        $per_page = 1;
+
+        $result = $this->endpoint->getTeamsForRetentionPolicy($policy_id, $page, $per_page);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$result'. (undefined)

(UnusedLocalVariable)